tests/python/FixedFunctionTransformTest.py

```diff
@@ -60,8 +60,7 @@ def test_style(self):
 
         for style in OCIO.FixedFunctionStyle.__members__.values():
             if style not in [OCIO.FIXED_FUNCTION_ACES_GAMUTMAP_02,
-                             OCIO.FIXED_FUNCTION_ACES_GAMUTMAP_07,
-                             OCIO.FIXED_FUNCTION_ACES_GAMUTMAP_13]:
+                             OCIO.FIXED_FUNCTION_ACES_GAMUTMAP_07]:
                 self.tr.setStyle(style)
                 self.assertEqual(self.tr.getStyle(), style)
 
```

Comment:
Would be nice to do a bit more Python testing, for example that it's possible to get and set the 7 parameters and that the validation works.

Reply:
Added a few test cases as suggested.